client/package.json

```diff
@@ -14,15 +14,16 @@
   "devDependencies": {
     "@sentry/webpack-plugin": "^1.11.1",
     "@types/jest": "^26.0.0",
-    "@typescript-eslint/eslint-plugin": "^3.3.0",
-    "@typescript-eslint/parser": "^3.3.0",
+    "@typescript-eslint/eslint-plugin": "^3.6.0",
```

Comment:
make sure you always reinstall the linter after changing package.json.
eslint-plugin-import is too new of a version